### ISSUE TITLE
When there are no fixture/seed data, `/` route raises exception

### ISSUE THREAD

**Issue by kljensen** [kljensen/yale-class-chat#23]
Loading index controller/view without fixture data create the following exception

```
app_1  | 21:47:41.361 request_id=FfwQVs9DuM3Bqs4AAAHx [info] GET /
app_1  | 21:47:44.481 request_id=FfwQVs9DuM3Bqs4AAAHx [info] Sent 404 in 3150ms
app_1  | 21:47:44.481 request_id=FfwQVs9DuM3Bqs4AAAHx [info] Converted error Ecto.NoResultsError to 404 response
```

**Comment by kljensen** [comment]
This is likely due to `uid` being NOT null and then using the following function

```
App.Accounts.get_user_by!(net_id)
```

**Comment by nperanzi** [comment]
Did you reload the page when you were previously logged on? I have had this error if I manually delete my user record while logged in, then refresh the page.

If you log out, delete fixtures, then reload, do you still get the error?

If only the former, is this worth fixing at the moment? Since the only scenario where this would arise is if we bork the database while users are logged in, which would have bigger issues than users getting 404 errors.

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by kljensen** [comment]
Nick - I think it is worth fixing. Yes, what you say will make it go away.
It is a 1 line fix I think.
Sincerely,
Kyle

**Comment by nperanzi** [comment]
Sounds good! Fixed in 5430ec70dd9997d37b2fa94329c592a2dc24a656
